lib/partition_setup.pm

```diff
@@ -0,0 +1,124 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+package partition_setup;
+
+use base Exporter;
+use Exporter;
+
+use strict;
+use testapi;
+
+our @EXPORT = qw(wipe_existing_partitions addpart addlv);
+
+my %role = qw(
+  OS alt-o
+  data alt-d
+  swap alt-s
+  raw alt-a
+);
+
+sub wipe_existing_partitions {
+    send_key $cmd{expertpartitioner};
+    assert_screen 'expert-partitioner';
+    wait_still_screen;
+    for (1 .. 4) {
+        send_key 'right';    # select vda hard disk
+    }
```

Comment:
when it will get unsafe then somebody will have to make "safe" all use of `for (1 .. n) {`